app/validate.py

```diff
@@ -0,0 +1,85 @@
+import re
+
+
+class Validation:
+    '''Class for returning validation results'''
+
+    def __init__(self):
+        self.status = None
+        self.message = ""
+
+
+def validate_register(json):
+    keys = ['email', 'first_name', 'last_name', 'password']
+    validation = Validation()
+    for field in keys:
+        if field not in json.keys():
+            validation.status = False
+            validation.message = "Missing fields in request data. Include: " +\
+                ", ".join(keys)
+            return validation
+
+    if len(json['password']) < 8:
+        validation.status = False
+        validation.message = "The password should be more than 8 characters."
+        return validation
+
+    match = re.match(
+        '^[_a-z0-9-]+(\.[_a-z0-9-]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})$',
```

Comment:
your current email validation does not work. Use https://regex101.com/ to test your regex expressions.